src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import cn from 'classnames';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  filterValue: Filter;
+  onClickFilter: (filterValue: Filter) => void;
+  deleteTodo: (todoId: number) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterValue,
+  onClickFilter,
+  deleteTodo,
+}) => {
+  const isCompleted = todos.some(todo => todo.completed);
+
+  const allNotCompleted = todos.filter(todo => !todo.completed).length;
+
+  const showFilterNavigation = Object.values(Filter).map(value => (
```

Comment:
You can just write it inside JSX (it's ok to do mapping inside JSX) instead of saving it into a variable